File: Linear Search/linearSearch.py
def linearSearch(arr, n, v):
    for i in range(0,n):
        if arr[i] == v:
            return i
    return -1

arr = [31, 41, 59, 26, 41, 58]
n = len(arr)
v = 26
search = linearSearch(arr, n, v)
if search == -1:
    print("No match found")
else:
    print(v,"found at position", search)

File: Linear Search/linearSearch.java
class linearSearch {
    public static int linSearch(int arr[], int v) {
        int n = arr.length;
        int i;
        for (i = 0; i < n; i++) {
            if (arr[i] == v) {
                return i;
            }
        }
        return -1;
    }

    public static void main(String args[]) {
        int arr[] = { 31, 41, 59, 26, 41, 58 };
        int v = 26;
        int search = linSearch(arr, v);
        if (search == -1) {
            System.out.println("No match found");
        } else {
            System.out.println(v + " found at position " + search);
        }
    }
}